peps/pep-8107.rst

```diff
@@ -0,0 +1,208 @@
+2025 election for the Python steering council, as specified in
+:pep:`13`. This is the steering council election for the 2026 term
+(i.e. Python 3.15).
+
+
+Election Administration
+=======================
+
+The steering council appointed the
+`Python Software Foundation <https://www.python.org/psf-landing/>`__
+Director of Infrastructure, Ee Durbin, to administer the election.
+
+
+Schedule
+========
+
+There will be a two-week nomination period, followed by a two-week
+vote.
+
+The nomination period will be: November 10, 2025 through `November 24, 2025 AoE
+<https://www.timeanddate.com/worldclock/fixedtime.html?msg=Python+Steering+Council+nominations+close&iso=20251125T00&p1=3399>`_ [#note-aoe]_.
+
+The voting period will be: November 28, 2025 through `December 12, 2025 AoE
+<https://www.timeanddate.com/worldclock/fixedtime.html?msg=Python+Steering+Council+voting+closes&iso=20251213T00&p1=3399>`_ [#note-aoe]_.
+
+
+Candidates
+==========
+
+Candidates must be nominated by a core team member. If the candidate
+is a core team member, they may nominate themselves.
+
+Nominees (in alphabetical order by first name):
+
+- TBD
+
+Withdrawn nominations:
+
+- None
+
+Voter Roll
+==========
+
+All active Python core team members are eligible to vote. Active status
+is determined as :pep:`described in PEP 13 <13#membership>`
+and implemented via the software at `python/voters <https://github.com/python/voters>`_
+[#note-voters]_.
+
+Ballots will be distributed based on the `Python Voter Roll
+<https://github.com/python/voters/blob/master/voter-files/>`_ [#note-voters]_
+for this election.
+
+While this file is not public as it contains private email addresses, the
+`Complete Voter Roll`_ by name will be made available when the roll is
+created.
+
+Election Implementation
+=======================
+
+The election will be conducted using the `BetterVoting
+<https://bettervoting.com>`__ service.
+
+.. attention::
+   This election will be the first where voting is conducted, and
+   results are calculated using
+   `Multi-winner Bloc STAR <https://www.starvoting.org/multi_winner>`__
+   as `approved by the core developers <https://discuss.python.org/t/64971>`__
+   and `codified <https://github.com/python/peps/pull/3958/files>`__
+   into :pep:`13`.
+
+
+Configuration
+-------------
+
+Create a `new election <https://bettervoting.com/new_election>`__.
+
+Poll or Election?: ``Election``
+
+Title?: ``2026 Python Steering Council Election``
+
+Restricted?: ``Yes``
+
+Contact Email: ``psf-election@python.org``
+
+Choose Voters: ``Email List``
+
+This will initialize the election and you will be forwarded to the election admin page.
+Further configuration is required.
+
+Click the pencil icon next to the election name on the admin.
+
+Election Description: ``Election for the Python steering council, as specified in PEP 13. This is steering council election for the 2026 term.``
+
+Enable Start/End Times?: ``Check this box``
+
+Time Zone: ``Midway Island, Samoa``
```

Comment:
I believe Midway Island (UTC-11) is an hour off from AoE (UTC-12), Baker and Howland Islands would work.

Reply:
Then please leave this open so that @ArendPeter can take the suggestion to add a time zone that _does_ reflect AoE?